webui/react/src/components/ProjectCard.tsx

```diff
@@ -1,104 +1,68 @@
   fetchProjects?: () => void;
   project: Project;
   showWorkspace?: boolean;
   workspaceArchived?: boolean;
 }
 
-const ProjectCard: React.FC<Props> = ({
-  project,
-  curUser,
-  fetchProjects,
-  workspaceArchived,
-  showWorkspace,
-}: Props) => {
-  const handleCardClick = useCallback(() => {
-    routeToReactUrl(paths.projectDetails(project.id));
-  }, [project.id]);
-
-  const users = Loadable.match(useUsers(), {
-    Loaded: (usersPagination) => usersPagination.users,
-    NotLoaded: () => [],
+const ProjectCard: React.FC<Props> = ({ project, fetchProjects, workspaceArchived }: Props) => {
+  const { menuProps, contextHolders } = useProjectActionMenu({
+    onComplete: fetchProjects,
+    project,
+    workspaceArchived,
   });
-  const user = users.find((user) => user.id === project.userId);
+
+  const classnames = [css.base];
+  if (project.archived) classnames.push(css.archived);
+
   return (
-    <ProjectActionDropdown
-      curUser={curUser}
-      project={project}
-      trigger={['contextMenu']}
-      workspaceArchived={workspaceArchived}
-      onComplete={fetchProjects}>
-      <div className={css.base} onClick={handleCardClick}>
-        <div className={css.nameRow}>
-          <h6 className={css.name}>
-            <Link inherit path={paths.projectDetails(project.id)}>
-              <Typography.Paragraph ellipsis={{ rows: 1, tooltip: true }}>
-                {showWorkspace && project.workspaceId !== 1 ? `${project.workspaceName} / ` : ''}
-                {project.name}
-              </Typography.Paragraph>
-            </Link>
-          </h6>
-          {project.archived && (
-            <Tooltip title="Archived">
-              <div>
-                <Icon name="archive" size="small" />
-              </div>
+    <Card
+      actionMenu={!project.immutable ? menuProps : undefined}
+      onClick={paths.projectDetails(project.id)}>
+      <div className={classnames.join(' ')}>
+        <Typography.Title className={css.name} ellipsis={{ rows: 5, tooltip: true }} level={5}>
+          {project.name}
+        </Typography.Title>
+        <div className={css.footer}>
+          <div className={css.experiments}>
+            <Tooltip
+              title={
+                `${project.numExperiments.toLocaleString()}` +
+                ` experiment${project.numExperiments === 1 ? '' : 's'}`
+              }>
+              <Icon name="experiment" size="small" />
+              <span>{nearestCardinalNumber(project.numExperiments)}</span>
             </Tooltip>
+          </div>
+          {project.archived ? (
+            <div className={css.archivedBadge}>Archived</div>
+          ) : (
+            project.lastExperimentStartedAt && (
+              <TimeAgo
+                className={css.lastExperiment}
+                datetime={project.lastExperimentStartedAt}
+                tooltipFormat="[Last experiment started ]MMM D, YYYY - h:mm a"
+              />
+            )
           )}
         </div>
```

Comment:
was there a reason user avatar was removed from the project cards? just want to make sure that was intentional, especially on the dashboard.

Reply:
Yes, that was an intentional decision from design. Feel free to discuss this with @michielpr